services/web/client/source/class/osparc/theme/Color.js

```diff
@@ -1,24 +1,10 @@
-/* ************************************************************************
-
-   osparc - the simcore frontend
-
-   https://osparc.io
-
-   Copyright:
-     2018 IT'IS Foundation, https://itis.swiss
-
-   License:
-     MIT: https://opensource.org/licenses/MIT
-
-   Authors:
-     * Tobias Oetiker (oetiker)
-     * Odei Maiz (odeimaiz)
-
-************************************************************************ */
-
 qx.Theme.define("osparc.theme.Color", {
```

Comment:
since you are using this as a mixin, maybe have `osparc.theme.mixin.Color`

Reply:
done!